pkg/deployment/reconcile/plan_builder_tls.go

```diff
@@ -514,7 +559,13 @@ func keyfileRenewalRequired(ctx context.Context,
 		}
 
 		// Verify AltNames
-		altNames, err := memberTls.GetServerAltNames(apiObject, spec, spec.TLS, service, group, member)
+		var altNames memberTls.KeyfileInput
+		if group.IsArangod() {
```

Comment:
Probably it's better to check for `group.IsArangosync()` instead, this way the old behavior will be applied to all non-sync members which might help avoiding some errors in future.